src/pages/onboarding/OnboardingPage.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Carousel from "nuka-carousel";
+
+import "./OnboardingPage.css"
+import ArrowToRightIcon from "../../images/path4.svg";
+
+import OnboardingService from '../../services/OnboardingService'
+
+type Props = {};
+
+type State = {
+    currentIndexCarousel: number
+}
+
+type OnboardingStep = {
+    title: string,
+    description: string,
+    imageSrc: string
+}
+
+const OnboardingSteps: OnboardingStep[] = [
+    {
+        title: 'Welcome to ArcWeather',
+        description: 'Your new favorite weather app, with all the features you need to have the best information available.',
+        imageSrc: 'assets/onboarding-image-01.png'
```

Comment:
Are these onboarding files on `/assets` or `/images`?

Reply:
These onboarding images are on /assets public static folder. 
What's the right way to do in this case, @arctouch-jonathansouza?